.wordlist.txt

```diff
@@ -124,4 +124,25 @@ AKS
 Workspaces
 Juju
 uptime
-IAM
\ No newline at end of file
+IAM
+Secureboot
+TPM
+UEFI
+cryptographically
+initialisation
+prebuilt
+keypair
+NitroTPM
+uefi
+github
```

Comment:
Is that a restriction of the spellchecker? because I prefer links in text when I only use the link once.
But I'll change it of course

Reply:
Using plain URLs will most like lead to spellcheck errors. So even if the link needs to be within the text (instead of the bottom of the page), it is better to use some markup text around it. 
Keeping links at the bottom of the page has a few advantages - the text (code) is more readable and it's easier to maintain the links.